test/fixtures/plugin-express.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+var TraceLabels = require('../../src/trace-labels.js');
+var shimmer = require('shimmer');
+var semver = require('semver');
+var methods = require('methods').concat('use', 'route', 'param', 'all');
+
+var SUPPORTED_VERSIONS = '4.x';
+
+module.exports = function(version, api) {
+  if (!semver.satisfies(version, SUPPORTED_VERSIONS)) {
```

Comment:
This plugin api work is an opportunity to change the flow of these functions if you see room for improvement. All plugins should be doing a supported versions check, maybe we could accept the plugins supported versions and do the check inside the agent so that every plugin doesn't have to depend on `semver` (and is guaranteed to do the check).